protos/flyteidl/core/workflow.proto

```diff
@@ -73,6 +73,9 @@ message NodeMetadata {
 
     // Number of retries per task.
     RetryStrategy retries = 5;
+
+    // Identify whether node is interruptible
+    bool interruptible = 6;
```

Comment:
I was going to ask you to add as similar flag to message but then I realized you already did:
https://github.com/lyft/flyteidl/blob/master/protos/flyteidl/admin/execution.proto#L191

I think it's important to think about precedence and how will these three work together.... please see my other comment for more details